docs/cloud-providers/azure/azure-provider.md

```diff
@@ -97,6 +97,7 @@ type: Opaque
 data:
   credentials: ${BASE64ENCODED_AZURE_ACCOUNT_CREDS}
 ---
+# Change the apiVersion below to azure.jet.crossplane.io/v1alpha1 if you are using provider-jet-azure 
```

Comment:
@ytsarev I am somewhat hesitant to make this change as this guide is mostly targeted to `provider-azure` (though the terminology is somewhat general). I would likely opt to try to move more of the getting started information to repos themselves -- these existing setup guides are somewhat a relic of the past and the potential for auth strategies to diverge between the two providers worries me from a maintenance perspective. Would you mind holding this change for the time-being until we get further clarity on https://github.com/crossplane/crossplane/pull/2930?

Reply:
@hasheddan As we don't merge the providers, can we merge this one in?